<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCDBDialog</class>
 <widget class="QDialog" name="SCDBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sidechains</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Which withdrawal do you ACK? (one per sidechain only!):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidgetVote">
         <property name="maximumSize">
          <size>
           <width>820</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Noto Mono</family>
          </font>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerDefaultSectionSize">
          <number>300</number>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Withdrawal Bundles</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTextBrowser" name="textBrowserSCDB">
           <property name="font">
            <font>
             <family>Noto Mono</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Sidechain withdrawal score history:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidgetHistory">
         <property name="font">
          <font>
           <family>Noto Mono</family>
          </font>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <property name="columnCount">
          <number>1</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Highlight item and use CTRL + C to copy text</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
